pavelib/utils/test/utils.py

```diff
@@ -17,7 +17,7 @@ def clean_test_files():
     """
     Clean fixture files used by tests and .pyc files
     """
-    sh("git clean -fqdx test_root/logs test_root/data test_root/staticfiles test_root/uploads")
+    sh("git clean -fqdx test_root/logs test_root/data test_root/uploads")
```

Comment:
Reviewers: I found it too irritating to have the static files deleted every time. This made it impossible to use fasttest to quickly re-run a test. Maybe we should find a way so that it cleans out the files only if fasttest is not set.